certification/internal/bindings/podman.go

```diff
@@ -0,0 +1,110 @@
+package bindings
+
+import (
+	"os"
+
+	"github.com/containers/podman/v3/pkg/bindings/containers"
+	"github.com/containers/podman/v3/pkg/bindings/images"
+	"github.com/containers/podman/v3/pkg/specgen"
+)
+
+func (p *PodmanClient) PullImage(nameOrID string) error {
+	_, err := images.Pull(p.Context, nameOrID, &images.PullOptions{})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (p *PodmanClient) SaveImage(nameOrID string) error {
+	outfile, err := os.Create("image.tar.gz")
```

Comment:
Can we at least name this with the nameOrID?

Reply:
What are you referring to as "this"? I have no reference point.

Reply:
image.tar.gz is an extremely generic name. That filename should be created based on nameOrID.

{nameOrID}.tar.gz